File: DotNet/CesiumLanguageWriter/Advanced/ICesiumHeightReferenceValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a height reference.
    /// </summary>
    public interface ICesiumHeightReferenceValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a height reference.
        /// </summary>
        /// <param name="value">The height reference.</param>
        void WriteHeightReference(CesiumHeightReference value);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumHeightReferenceValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;
import javax.annotation.Nonnull;

/**
 * A writer that can write a value as a height reference.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumHeightReferenceValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a height reference.
    * @param value The height reference.
    */
    void writeHeightReference(@Nonnull CesiumHeightReference value);
}